src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -6591,6 +6596,14 @@ void UpdateSysSizing(EnergyPlusData &state, Constant::CallIndicator const CallIn
                     state.dataSize->CalcSysSizing(AirLoopNum).RetHumRatAtCoolPeak = SysCoolRetHumRat;
                     state.dataSize->CalcSysSizing(AirLoopNum).OutTempAtCoolPeak = SysCoolOutTemp;
                     state.dataSize->CalcSysSizing(AirLoopNum).OutHumRatAtCoolPeak = SysCoolOutHumRat;
+                    if (state.dataSize->SysSizPeakDDNum(AirLoopNum).SensCoolPeakDD == 0 && saveCoolTUSizingIndex > 0) {
+                        CoolDDNum = state.dataSize->TermUnitFinalZoneSizing(saveCoolTUSizingIndex).CoolDDNum;
+                        CoolTimeStepNum = state.dataSize->TermUnitFinalZoneSizing(saveCoolTUSizingIndex).TimeStepNumAtCoolMax;
+                        state.dataSize->SysSizPeakDDNum(AirLoopNum).SensCoolPeakDD = CoolDDNum;
+                        state.dataSize->SysSizPeakDDNum(AirLoopNum).cSensCoolPeakDDDate = state.dataSize->DesDayWeath(CoolDDNum).DateString;
+                        state.dataSize->SysSizPeakDDNum(AirLoopNum).TimeStepAtSensCoolPk(HeatDDNum) = CoolTimeStepNum;
+                        state.dataSize->CalcSysSizing(AirLoopNum).CoolDesDay = state.dataSize->SysSizing(CoolDDNum, AirLoopNum).CoolDesDay;
+                    }
```

Comment:
When entering the outter IF block the peak sizing data is updated. However, the peak day/time is what was found for coincident sizing (or blank if there was no coincident peak). This change uses the peak day/time of the last zone in the loop as the reported day/time of non-coincident peak.

Reply:
Typo on line 6604 using HeatDDNum instead of CoolDDNum. When testing results for always printing this choice of non-coincident peak reporting (i.e., removing SensCoolPeakDD == 0) this became apparent. Once corrected, the system non-coincident cooling peak day/time matched the last zones peak day/time (previously showed 16:00 for system peak).

    Zone Sizing Information, SPACE5-1, Cooling, 1168.36825, 1168.36825, 9.82782E-002, 0.13906, CHICAGO_IL_USA ANNUAL COOLING 1% DESIGN CONDITIONS DB/MCWB, 7/21 15:00:00
    System Sizing Information, VAV SYS 1, Cooling, Sensible, 11210.35, 0.77212, 0.77212, CHICAGO_IL_USA ANNUAL COOLING 1% DESIGN CONDITIONS DB/MCWB, 7/21 15:00:00